stl/inc/ranges

```diff
@@ -904,8 +927,7 @@ namespace ranges {
         inline constexpr empty_view<_Ty> empty;
     } // namespace views
 
-    template <copy_constructible _Ty>
-        requires is_object_v<_Ty>
+    template <_Valid_movable_box_object _Ty>
     class single_view : public view_interface<single_view<_Ty>> {

```

Comment:
Below, `single_view(const T& t)` is supposed to gain the constraint `requires copy_constructible<T>`.